Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/TestData/ExamplePBR/example_pbr2.materialcanvasnode.azasset

```diff
@@ -0,0 +1,148 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "DynamicNodeConfig",
+    "ClassData": {
+        "category": "Material Templates",
+        "title": "Example PBR 2",
+        "settings": {
+            "templatePaths": [
+                "@gemroot:MaterialCanvas@/Assets/MaterialCanvas/TestData/ExamplePBR/MaterialGraphName_ForwardPass.azsl.template",
+                "@gemroot:MaterialCanvas@/Assets/MaterialCanvas/TestData/ExamplePBR/MaterialGraphName_ForwardPass.shader.template",
+                "@gemroot:MaterialCanvas@/Assets/MaterialCanvas/TestData/ExamplePBR/MaterialGraphName.materialtype.template",
+                "@gemroot:MaterialCanvas@/Assets/MaterialCanvas/TestData/ExamplePBR/MaterialGraphName.material.template"
+            ]
+        },
+        "inputSlots": [
+            {
+                "name": "inPositionOffset",
+                "displayName": "Position Offset",
+                "description": "Position Offset",
+                "supportedDataTypes": [
+                    "float",
+                    "float2",
+                    "float3",
+                    "float4",
+                    "color"
+                ],
+                "defaultValue": {
+                    "$type": "Vector4",
+                    "Value": [
+                        0.0,
+                        0.0,
+                        0.0,
+                        0.0
+                    ]
+                },
+                "settings": {
+                    "instructions": [ "SLOTTYPE SLOTNAME = SLOTVALUE;" ]
+                }
+            },
+            {
+                "name": "inBaseColor",
+                "displayName": "Base Color",
+                "description": "Base Color",
+                "supportedDataTypes": [
```

Comment:
Suggesting excluding `float4` since "alpha" would be handled separately as part of the future transparency supported lighting model.